pages/news/2020-07-08-intro-outro-sections.html

```diff
@@ -0,0 +1,106 @@
+title: Intro & Outro Cues, Silence Detection, and AutoDJ Improvements
+author: Be.
+date: 2020-07-08 01:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown:"extra" %}
+
+Do you use hotcues to mark the points you use to mix in and out of tracks? Have you wished AutoDJ knew to mix in and out at those points?
+In Mixxx 2.3, we have introduced new intro & outro cues for these purposes.
```

Comment:
```suggestion
In Mixxx 2.3, we'll introduce new intro & outro cues for these purposes.
```

Or something similar, so that it doesn't sound like 2.3 has already been released.